plugins/css-blank-pseudo/src/browser.js

```diff
@@ -108,39 +114,71 @@ export default function cssBlankPseudoInit(opts) {
 
 	const handler = generateHandler(options.replaceWith);
 
-	document.body.addEventListener('change', handler);
-	document.body.addEventListener('input', handler);
-
-	// observe value changes on <input>, <select>, and <textarea>
-	const window = (document.ownerDocument || document).defaultView;
+	if (document.body) {
+		document.body.addEventListener('change', handler);
+		document.body.addEventListener('input', handler);
+	} else {
+		window.addEventListener('load', () => {
+			if (document.body) {
+				document.body.addEventListener('change', handler);
+				document.body.addEventListener('input', handler);
+			}
+		});
+	}
 
-	observeValueOfHTMLElement(window.HTMLInputElement, handler);
-	observeValueOfHTMLElement(window.HTMLSelectElement, handler);
-	observeValueOfHTMLElement(window.HTMLTextAreaElement, handler);
-	observeSelectedOfHTMLElement(window.HTMLOptionElement, handler);
+	observeValueOfHTMLElement(self.HTMLInputElement, handler);
+	observeValueOfHTMLElement(self.HTMLSelectElement, handler);
+	observeValueOfHTMLElement(self.HTMLTextAreaElement, handler);
+	observeSelectedOfHTMLElement(self.HTMLOptionElement, handler);
 
 	// conditionally update all form control elements
 	Array.prototype.forEach.call(
-		document.querySelectorAll(BLANK_CANDIDATES),
+		document.querySelectorAll('input, select, textarea'),
 		node => {
 			handler({ target: node });
 		},
 	);
 
-	if (typeof window.MutationObserver !== 'undefined') {
+	if (typeof self.MutationObserver !== 'undefined') {
 		// conditionally observe added or unobserve removed form control elements
 		new MutationObserver(mutationsList => {
 			mutationsList.forEach(mutation => {
 				Array.prototype.forEach.call(
 					mutation.addedNodes || [],
 					node => {
-						// Matches isn't polyfilled here since IE9 doesn't have support for MO
-						if (node.nodeType === 1 && node.matches(BLANK_CANDIDATES)) {
+						if (node.nodeType === 1 && isFormControlElement(node)) {
 							handler({ target: node });
 						}
 					},
 				);
 			});
 		}).observe(document, { childList: true, subtree: true });
+	} else {
+		window.addEventListener('load', () => {
```

Comment:
If the init is called in the html head we miss the initial state.
Adding a few event callbacks to make sure the initial state is correct even if we have no support for mutation observer